From: Kill Your TV <killyourtv@i2pmail.org>
Date: Fri, 3 Oct 2014 18:52:52 +0000
Subject: default tor proxy

Since my packages are generally used by I2P/Tor users, they'd probably benefit
by -p defaulting to Tor.
---
 config.def.h | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

--- a/config.def.h
+++ b/config.def.h
@@ -16,8 +16,8 @@
 static int      tcp         = 0;
 static int      proxy       = 0;
 static int      proxytype   = 2; /* 1 = HTTP, 2 = SOCKS5 */
-static char     proxyaddr[] = "localhost";
-static uint16_t proxyport   = 8080;
+static char     proxyaddr[] = "127.0.0.1";
+static uint16_t proxyport   = 9050;
 
 static struct node nodes[] = {
 	{
